circuits/circuits/send.circom

```diff
@@ -99,8 +102,7 @@ template Validate(levels) {
 
     signal output is_send_valid;
 
-    signal is_signature_valid <== 0; // TODO
-    signal signer <== 0; // TODO: recover signer from signature, or just pass it in as input
+    signal is_signature_valid === 1; // TODO: ensure in smart contract's force_include() fn that signature is valid, otherwise revert.
```

Comment:
You mean we should check the signature in the solidity? It has to be checked in the circuit anyway, because most transactions don't use force_include, and their signatures need to be checked in the circuit. So I'm not sure why we'd want to check the signature in solidity.

Reply:
need to fail on invalid signature instead of returning initial_root. this ensures that the operator cannot by pass force inclusions by passing an invalid sig.